scripts/test-vue3.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+yarn remove vue vue-jest vue-template-compiler babel-preset-es2015
+yarn add vue@3.1.2 @vue/server-renderer@3.1.2 vue-jest@5.0.0-alpha.10 vue-loader@16.1.2 jest@26.6.3 babel-jest@25.2.6 @babel/preset-env@7.14.5 @babel/plugin-transform-runtime@7.14.5 -D -E
+echo "export * from './index-3';" > src/util/vue-compat/index.js
+rm .babelrc
```

Comment:
ah it's because the default is babelrc, could we use .config.js everywhere?

Reply:
probably? but I got hit by an issue when using babel.config.js with vue 2:

```
import createSerializer from 'jest-serializer-html/createSerializer';                                                                                          ^^^^^^
    SyntaxError: Cannot use import statement outside a module
```

It works with `.babelrc` but not with `.config.js`.

However, in vue 3, it works only with `.config.js`. haha...